### PR TITLE
Ensure passed non-string trees are handled

### PR DESCRIPTION
**Pull request by chadhietala**
I think this closes #35?

### CODE REVIEW

**Comment by stefanpenner** [comment]
nice

**Comment by stefanpenner** [comment]
cut a release? I can if you don't have time.

**Comment by chadhietala** [comment]
done.

File: lib/find.js [hunk 1]
```diff
@@ -102,7 +102,13 @@ function _find(tree, _options) {
   var options, root;
 
   if (arguments.length === 1) {
-    var decomposition = decompose(tree);
+    var decomposition;
+
+    if (typeof tree === 'object') {
+      decomposition = decompose(tree.inputTree);
```
Comment (rwjblue) [inline lib/find.js]: There is nothing in broccoli land that requires the tree be stashed at `inputTree`.  What if the tree we are provided is nested two levels?  Using this technique we'd have to then do `tree.inputTree.inputTree` (assuming all of the levels use the common name of `inputTree`).

Comment (stefanpenner) [inline lib/find.js]: hmm, should we.

1. support that convention, an handle this recursive?
2. is their another mechanism to accomplish this.

Comment (rwjblue) [inline lib/find.js]: I'm not sure I fully understand why this was required, so I wasn't sure what to suggest as an alternative.

Comment (chadhietala) [inline lib/find.js]: How I feel a lot of times with Broccoli:


Anyways I think what we need to do is check if the first argument is an object, if is we just use it as the root for the funnel passing the decompose options.

**Comment by stefanpenner** [comment]
Ya we have to keep polishing that broccoli. (And our grasp on it)

**Pull request merged**